docs/getting-started/getting-started.mdx

```diff
@@ -0,0 +1,11 @@
+---
+title: 'Getting Started with Conduit'
+sidebar_position: 0
+sidebar_label: "Getting Started"
+---
+
+import DocCardList from '@theme/DocCardList';
+
+To help you get started, here are some of the most popular pages in our documentation:
+
+<DocCardList/>
```

Comment:
One thing that bothers me a bit is that some expandable pages are clickable, while others aren't (I mean the pages that contain nested pages). I would love it if we could consolidate that, so it's consistent. WDYT?

Reply:
@lovromazgon I agree. That's also something that I wanted to change. I was considering only changing the pages I touched, but I can squeeze that change in as well.